internal/store/statefulset_test.go

```diff
@@ -65,6 +65,7 @@ func TestStatefulSetStore(t *testing.T) {
 				# HELP kube_statefulset_metadata_generation [STABLE] Sequence number representing a specific generation of the desired state for the StatefulSet.
 				# HELP kube_statefulset_persistentvolumeclaim_retention_policy Count of retention policy for StatefulSet template PVCs
 				# HELP kube_statefulset_replicas [STABLE] Number of desired pods for a StatefulSet.
+                # HELP kube_statefulset_ordinals_start Start ordinal of the StatefulSet.
```

Comment:
Looks like indentation here is off

Reply:
Thanks for catching (tabs vs. spaces issue). Fixed here and on line 371.